src/general/PauseMenu.cs

```diff
@@ -27,11 +27,21 @@ public class PauseMenu : ControlWithInput
     [Export]
     public NodePath LoadSaveListPath;
 
+    [Export]
+    public NodePath ExitConfirmationDialogPath;
```

Comment:
I'd rename anything with "ExitConfirmation" to "UnsavedProgressWarning" that way the last few bits I find similar to the old PR would be gone.